elescope.el

```diff
@@ -45,6 +45,14 @@ Defaults to 1 which makes all clones shallow clones."
   :group 'eslescope
   :type 'integer)
 
+
+(defcustom elescope-use-namespace nil
+  "Use project username as namespace for path.
+
+Default as nil"
```

Comment:
Could we have an example in the description? It took me a bit of time to understand the functionality, but now I'm all aboard. Maybe something like

```elisp
(defcustom elescope-use-full-path nil
"Use the full project path for the resulting clone.

If non-nil, use the full project path including
username/organisation to clone: cloning 'john/foo' and 'john/bar'
results in two 'foo' and 'bar' clones inside a parent 'john'
folder, as opposed to the default, flat hierarchy of 'foo' and
'bar'."
  :group 'elescope
  :type 'boolean)
```

Feel free to use your own style, as long as simple minds like me can understand it.

Reply:
I thought that the `namespace` might make it easy to recognize what I mean. And my English just poor enough🤣. Already replace it.